framework/PostProcessors/BasicStatistics.py

```diff
@@ -967,22 +974,29 @@ def sensitivityCalculation(self,featVars, targVars, featSamples, targSamples, in
       @ In, intersectionSet, boolean, True if some target variables are in the list of features
       @ Out, da, xarray.DataArray, contains the calculations of sensitivity coefficients
     """
-    if not intersectionSet:
-      senMatrix = LinearRegression().fit(featSamples,targSamples).coef_
+    if self.multipleLinear:
+      if not intersectionSet:
+        senMatrix = LinearRegression().fit(featSamples,targSamples).coef_
```

Comment:
comments/...add explain what it is done...

Reply:
added.